src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java

```diff
@@ -117,6 +117,7 @@ private KafkaConfig createConfig()
         config.put(KafkaConfig.DeleteTopicEnableProp(), "false");
         config.put(KafkaConfig.ControlledShutdownRetryBackoffMsProp(), "100");
         config.put(KafkaConfig.LogCleanerDedupeBufferSizeProp(), "2097152");
+        config.put(KafkaConfig.OffsetsTopicReplicationFactorProp(), (short) 1);
```

Comment:
2, or maybe even 3, are much better than 1.  1 is almost never going to be what the user wants.  We should add some documentation of this too so users of this library won't be as surprised.  Isn't it a pain to change replication after topic creation?